### PR TITLE
#444 - GetUsersSlice introduced

### PR DESCRIPTION
**Pull request by olenagerasimova**
For  #444 implemented `GetUsersSlice` to obtain users list with urls.

### CODE REVIEW

File: src/main/java/com/artipie/api/artifactory/GetUsersSlice.java [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+                                );
+                                return new RsJson(json);
+                            }
+                        )
+                    ).orElse(CompletableFuture.completedFuture(StandardRs.NOT_FOUND))
+                )
+            );
+        } catch (final IOException err) {
+            Logger.error(this, err.getMessage());
+            res = new RsWithStatus(RsStatus.INTERNAL_ERROR);
```
Comment (olegmoz) [inline src/main/java/com/artipie/api/artifactory/GetUsersSlice.java:92]: @olenagerasimova I'd suggest to not translate error to response here, as it is done on upper level too. You could rethrow `IOException` wrapped into runtime exception or return `Response` which returns failed future on `send(...)` call

File: src/main/java/com/artipie/api/artifactory/GetUsersSlice.java [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+        }
+        return res;
+    }
+
+    /**
+     * Returns json for user.
+     * @param name Username
+     * @param base Base url
```
Comment (olegmoz) [inline src/main/java/com/artipie/api/artifactory/GetUsersSlice.java:100]: @olenagerasimova parameters order got me confused, could you please put `base` before `name`?

**Comment by olenagerasimova** [comment]
@olegmoz thanks, done

**Comment by 0crat** [comment]
Job `gh:artipie/artipie#476` is not assigned, can't get performer

**Pull request merged**